product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -1,32 +1,44 @@
-| Project      | Failed maintenance upgrade on a cluster                                                          | Project owner/editor             | All                  |
-| Project      | Paused cluster will automatically reactivate in 24 hours                                         | Project owner/editor             | All                  |
-| Project      | Paused cluster was automatically reactivated                                                     | Project owner/editor             | All                  |
-| Project      | You must set up the encryption key permission for your CMK-enabled cluster                       | Project owner/editor             | All                  |
-| Project      | Key error with CMK-enabled cluster                                                               | Project owner and project editor | All                  |
-| Project      | User is invited to a project (displays only to the Project owner)                                | Project owner                    | All                  |
-| Project      | New role is assigned to you                                                                      | Account owner                    | All                  |
-| Project      | Role is unassigned from you                                                                      | Account owner                    | All                  |
-| Project      | Failed connection to third-party monitoring integration (and future non-monitoring integrations) | Project owner/editor             | All                  |
+Different types of events are sent as notifications. Users with different roles can configure the preferences to receive these notifications in the in-app inbox, by email, or both. 
+The notifications are categorized into following preference sections:
+- Account
+- Organizations
+- Projects
+
+The notifications under **Account** preference section:
+
+- Project role unassigned from you
+- Personal access key is about to expire
+- New project role assigned to you
+
+The notifications under **Organizations** preference section:
+
+- Payment method added (specific to organizations that opted credit card payment option)
+- Machine user API key is expiring
 
 !!!note
-Under "Subscription type," "All" means digital self-service, direct purchase, and Azure Marketplace. For more information, see [subscription types](/biganimal/latest/pricing_and_billing/#payments-and-billing).
+This section is visible only to the organization owner. If the current user is owner of more than one organization, then this section lists the preferences for all the organizations.
 !!!
 
+The notifications under **Projects** preference section:
+
+- Paused cluster reactivated                                                     
+- Upcoming cluster maintenance upgrade                                            
+- CMK error                                                               
```

Comment:
We should expand the `CMK` here to `Customer-Managed Key` @neel5481 can you provide better description of it here, as I am worrying user reading this line might still feel confused.

Reply:
Below notification needs some improvement as user is not able to understand it.

Change as below or some relevant text.

`CMK error`  can be changed to any one as below

 Pause the cluster due to encryption key error      OR
 Encryption key error and pause the cluster


`User encryption key permission for cluster` can be changed to any one as below

Provide encryption key permission during cluster provision   OR
Encryption key permission and cluster provision